lib/src/components/NxDrawer/__tests__/NxDrawer.test.tsx

```diff
@@ -201,21 +244,100 @@ describe('NxDrawer', function() {
       expect(screen.getByText('Title')).toBeInTheDocument();
     });
 
-    it('renders close button', async function() {
-      quickRender({
-        children: (
-          <NxDrawer.Header>
-            <NxDrawer.HeaderTitle>Title</NxDrawer.HeaderTitle>
-          </NxDrawer.Header>
-        )
+    describe('renders a close button', function() {
+
+      describe('with a default tooltip', function() {
+        it('if closeBtnTooltip prop is not provided', async function() {
+          const user = userEvent.setup();
+
+          quickRender({
+            children: (
+              <NxDrawer.Header>
+                <NxDrawer.HeaderTitle>Title</NxDrawer.HeaderTitle>
+              </NxDrawer.Header>
+            )
+          });
+
+          const dialog = screen.getByRole('dialog', { hidden: true });
+          await fireEvent.animationEnd(dialog);
+
+          const closeButton = screen.getByRole('button', { name: /close/i });
+
+          expect(closeButton).toBeInTheDocument();
+
+          await runTimers();
+          expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
+
+          await user.hover(closeButton);
+          await runTimers();
+
+          const tooltip = screen.getByRole('tooltip');
+
+          expect(tooltip).toHaveTextContent('Close');
+        });
+
+        it('if only the closeBtnDisabled prop is provided', async function() {
```

Comment:
This test seems unnecessary now that the `closeBtnDisabled` and `closeBtnTooltip` props operate completely independently

Reply:
> This test seems unnecessary now that the `closeBtnDisabled` and `closeBtnTooltip` props operate completely independently

I must've forgotten to clean up this one :sweat_smile:
https://github.com/sonatype/sonatype-react-shared-components/pull/1169/commits/bd776179008033dd4a5e3db385b1cf0156dabeae